analyzer/build.gradle.kts

```diff
@@ -14,13 +14,10 @@ gradlePlugin {
 }
 
 dependencies {
-    /*compile("com.fasterxml.jackson.core:jackson-databind:2.9.8")
```

Comment:
Ah, yes, wanted to clean that up but then forgot… 😞

Reply:
No worries. I didn't comment on your PR because I knew I was going to make changes anyway